core/src/main/java/com/datastrato/gravitino/tag/TagManager.java

```diff
@@ -22,32 +32,119 @@ public class TagManager {
 
   private static final Logger LOG = LoggerFactory.getLogger(TagManager.class);
 
-  public TagManager(IdGenerator idGenerator, EntityStore entityStore) {}
+  private final IdGenerator idGenerator;
+
+  private final EntityStore entityStore;
+
+  public TagManager(IdGenerator idGenerator, EntityStore entityStore) {
+    if (entityStore instanceof KvEntityStore) {
+      LOG.warn(
```

Comment:
I'm afraid the warning level is not enough. Furthermore, the error messages are consistent. Can you unify them?

Reply:
What is the meaning of `the error messages are consistent`?

Reply:
I mean the error messages are the same here.